modules/input.nix

```
@@ -433,77 +431,77 @@ in
         Keyboard layouts to use.
       '';
     };
-    numlockOnStartup = mkOption {
-      type = with types; nullOr (enum numlockSettings);
+    numlockOnStartup = lib.mkOption {
+      type = with lib.types; nullOr (enum numlockSettings);
       default = null;
       example = "on";
       description = ''
         Numpad settings at startup.
       '';
     };
-    repeatDelay = mkOption {
-      type = with types; nullOr (ints.between 100 5000);
+    repeatDelay = lib.mkOption {
+      type = with lib.types; nullOr (ints.between 100 5000);
       default = null;
       example = 200;
       description = ''
         How many milliseconds you need to hold a key down before the input
         starts repeating.
       '';
     };
-    repeatRate = mkOption {
-      type = with types; nullOr (numbers.between 0.2 100.0);
+    repeatRate = lib.mkOption {
+      type = with lib.types; nullOr (numbers.between 0.2 100.0);
       default = null;
       example = 50.0;
       description = ''
         How quick the inputs should be repeated when holding down a key.
       '';
     };
-    options = mkOption {
-      type = with types; nullOr (listOf str);
+    options = lib.mkOption {
+      type = with lib.types; nullOr (listOf str);
       default = null;
       example = [
         "altwin:meta_alt"
         "caps:shift"
-        "custom:types"
+        "custom:lib.types"
```

This shouldn't change.
```suggestion
        "custom:types"
```

---

Whoops! `rg` was a bit too eager there. Resolved.